src/main/java/io/jmix/uisamples/bean/OverviewPageGenerator.java

```diff
@@ -0,0 +1,233 @@
+            verticalLayout.setPadding(false);
+            verticalLayout.setMaxWidth("41em");
+            for (Element textElement : textElements) {
+                verticalLayout.add(createLabel(textElement, messagesPrefix));
+            }
+
+            overviewRoot.add(verticalLayout);
+        }
+    }
+
+    private void initSamples(Element samples, VerticalLayout overviewRoot, String messagesPrefix) {
+        if (samples != null) {
+            FlexLayout flexLayout = uiComponents.create(FlexLayout.class);
+            flexLayout.setAlignContent(FlexLayout.ContentAlignment.START);
+            flexLayout.setFlexDirection(FlexLayout.FlexDirection.ROW);
+            flexLayout.setJustifyContentMode(!isSmallDevice()
+                    ? FlexComponent.JustifyContentMode.START
+                    : FlexComponent.JustifyContentMode.CENTER);
+            flexLayout.setFlexWrap(FlexLayout.FlexWrap.WRAP);
+            flexLayout.addClassName(LumoUtility.Gap.LARGE);
+
+            overviewRoot.add(flexLayout);
+
+            List<Element> samplesList = samples.elements();
+            for (Element sample : samplesList) {
+                VerticalLayout verticalLayout = uiComponents.create(VerticalLayout.class);
+                verticalLayout.setPadding(false);
+                verticalLayout.setWidth("20em");
+                verticalLayout.add(createImage(sample.element("image")));
+                verticalLayout.addClassName(LumoUtility.Gap.MEDIUM);
+
+                List<Element> textElements = sample.elements("text");
+
+                for (Element textElement : textElements) {
+                    verticalLayout.add(createLabel(textElement, messagesPrefix));
+                }
+
+                List<Element> tagElements =  sample.elements("tag");
+                if (!tagElements.isEmpty()) {
+                    verticalLayout.add(createTags(tagElements));
+                }
+
+                flexLayout.add(verticalLayout);
+            }
+        }
+    }
+
+    private Component createTags(List<Element> tagElements) {
+        FlexLayout flexLayout = uiComponents.create(FlexLayout.class);
+        flexLayout.setAlignContent(FlexLayout.ContentAlignment.START);
+        flexLayout.setFlexDirection(FlexLayout.FlexDirection.ROW);
+        flexLayout.setJustifyContentMode(FlexComponent.JustifyContentMode.START);
+        flexLayout.setFlexWrap(FlexLayout.FlexWrap.WRAP);
+        flexLayout.addClassName(LumoUtility.Gap.SMALL);
+
+        for (Element tagElement : tagElements) {
+            Component tag = createComponentByName(tagElement.attributeValue("component"));
+
+            tag.getElement().getThemeList().add("badge pill");
+            addClassNames(tag, tagElement.attributeValue("classNames"));
+            if (tag instanceof HasText hasText) {
+                hasText.setText(tagElement.attributeValue("text"));
+            }
+            tag.addClassName(LumoUtility.FontWeight.LIGHT);
```

Comment:
This doesn't work because the `[theme]` styles is higher in the style hierarchy. You will have to use styles using `.css`. 

This line will have to be deleted because it doesn't work anyway.